machina/templates/machina/partials/posted_by.html

```diff
@@ -0,0 +1,16 @@
+{% load i18n %}
+
+{% if display_label_by == None or display_label_by %}
+    {% trans "By:" %}
+{% endif %}
+{% if poster %}
+    {% include "partials/profile_link.html" %}
+{% else %}
+    {% if username_highlighted %}<b>{% endif %}{{ username }}{% if username_highlighted %}</b>{%endif%}
+{% endif %}
```

Comment:
This block is not i18n-friendly: the "By:" string is inserted as a prefix whereas in the previous version of the code this was fully handled as a proper translation with variable substitutions.

Reply:
Thank you for you feedback, I rewrote the partial and didn't cut any translation.